ui/intermittent-failures/BugDetailsView.jsx

```diff
@@ -139,14 +137,17 @@ const BugDetailsView = props => {
         <React.Fragment>
           <Row>
             <Col xs="12" className="text-left">
-              <Link to={lastLocation || '/'}>
-                <FontAwesomeIcon
-                  icon={faArrowLeft}
-                  className="mr-1"
-                  title="go to intermittent failures view main page"
-                />
-                back
-              </Link>
+            <Breadcrumb>
+                <BreadcrumbItem>
+                  <Link title="Intermittent Failures View" to={'/'}>Treeherder</Link>
```

Comment:
Treeherder is the parent of this app, so you don't want to have it redirect to '/' it should redirect to 'https://treeherder.mozilla.org/' here and in the other breadcrumb. The titles should also change to "Treeherder home page" or something like that.

Reply:
Thanks Sarah. Made those changes.